### ISSUE TITLE
Issue on ConfuserEx 0.60

### ISSUE THREAD

**Issue by Deltafox79** [cawk/ConfuserEx-Unpacker#5]
Hi, I have a problems whit correct un pack ConfuserEx 0.60..

This is the target : 

https://www4.zippyshare.com/v/9CwDiznW/file.html

i try also whit ollyDbg whitout corret obfuscation..

Any help on this?

best regards and Thank's

**Comment by RetiredQQ** [comment]
It perfectly works on my side. First I try use ConfuserEx Unpacker after that use de4dot.

**Comment by RetiredQQ** [comment]
No, since I was using official version too with other CC tools.

**Comment by RetiredQQ** [comment]
I use static options to unpack it.
